openproblems/tasks/spatial_decomposition/methods/cell2location.py

```diff
@@ -0,0 +1,20 @@
+from ....tools.decorators import method
+from ....tools.normalize import log_cpm
+from ....tools.utils import check_version
+
+# import cell2location
+
+
+@method(
+    method_name="Cell2Location",
+    paper_name="Comprehensive mapping of tissue cell architecture via integrated single cell and spatial transcriptomics",  # noqa: E501
+    paper_url="https://www.biorxiv.org/content/10.1101/2020.11.15.378125v1",
+    paper_year=2020,
+    code_url="https://github.com/BayraktarLab/cell2location"
+    "sklearn.neural_network.MLPClassifier.html",
+    code_version=check_version("cell2location"),
+)
+def cell2location_log_cpm(adata):
+    log_cpm(adata)
+    # do something
```

Comment:
Like what 😉?

Reply:
this was supposed to be added in the hackathon but eventually got delayed. Shall we remove it?

Reply:
Bump

Reply:
It should be but I don't fully understand how the full workflow is supposed to operate. I would appreciate a short zoom chat.

Reply:
Either this is added now, or we move it to a separate PR. @giovp @LuckyMD @vitkl thoughts?